### DISCUSSION TITLE
Alternative to tabs for main UI

### DISCUSSION THREAD

**Discussion by harshad1** [gsantner/markor#1586] [Ideas] [answered]
The tabs currently used in the main UI have 2 problems:

1. They take up a lot of vertical space
2. They sit between the keyboard and the text actions, leading to accidental clicks.

Some possible fixes include:
1. Hiding the tabs when the keyboard is open
2. Reducing the height of the tabs
3. Moving the tabs to the top
4. Some completely new UI approach

@gsantner Any thoughts on this?

**Comment by gsantner** [comment, marked as answer]
* Most important for me is - that the filebrowser/todo/quicknote/the rest is just one finger press (or swipe) away.  
* That those are reachable single handed (top bar = not single hand anymore, due the size of phones nowadays is rather what was considered a tablet in the past). 
* Generally I prefer static positions, no automatic hiding magic (which again does requires **some** user interaction to move on to the other of filebrowser/todo/quicknote)
* I see the main tabs as "quick thing to get the job done". For full capabilities and editing, users should open the dedicated editor activity. Where no additional such bar/whatever is present - and thus more vertical space

sounds awful lot like "make it better but keep it still old" hehe, well. I yet have not discovered any better option than bottom tabs which would better fit the flow :smile:

**Reply by harshad1** [reply]
> I yet have not discovered any better option than bottom tabs which would better fit the flow

How do you feel about adding an option to move the tabs to the top? It would still be default-bottom

**Reply by gsantner** [reply]
Every option requires maintainance, comes at a cost. If it's to be changed, I don't want to introduce "two new versions/variants".

This should be a single one and well-thought

**Reply by gsantner** [reply]
In the past I had this ... with swipeable tabs, on top

![grafik](https://user-images.githubusercontent.com/6735650/153511674-ee187d42-065b-4deb-83ef-0fd0920af8b9.png)

however, it ended up this way after some time, and thats how it is now for some time 

![grafik](https://user-images.githubusercontent.com/6735650/153511651-ecfcede8-032a-4411-b8cd-5ad9ee31b264.png)
